functional-tests/faucet_test.go

```diff
@@ -41,7 +42,7 @@ func TestFaucetSendFunds(t *testing.T) {
 	blockTime := time.Second * 5
 
 	// Setup first node, note: Testbed.Name() is the directory
-	genesisTime := time.Unix(123456789, 0)
+	genesisTime := time.Now()
```

Comment:
We would like this to be deterministic. I'm assuming this is to debug CI. Unless I'm wrong, please revert before merging.

Reply:
Unfortunately the specced out timing rules mean that we either need to do this, mock out the clock, or modify the test to allow for a time-of-test dependent number of null blocks between genesis and the first non-null.  This seems like the best tradeoff for a functional test (if it were an integration test mocking out clock would probably be preferred).